### PR TITLE
Skeletonized RP Doc's

### PR DESCRIPTION
**Pull request by hkraemer**
Here is a first draft. Unfortunately I can run and try it myself, once JuliaDynamics/RecurrenceAnalysis.jl#128 is merged.

### CODE REVIEW

File: docs/src/rqa/rplots.md [hunk 1]
```diff
@@ -116,6 +116,50 @@ which justifies why recurrence plots are so fitting to be used in embedded times
...
+title("RP of monochromatic signal")
+
+subplot(122)
+Rg = grayscale(R_skel)
+imshow(Rg, cmap = "binary_r", extent = (1, size(R)[1], 1, size(R)[2]))
+title("skeletonized RP")
+fig.tight_layout(pad=0.3); fig
+savefig("skeleton1.png"); nothing # hide
+```
+![](skeleton1.png)
```
Comment (Datseris) [inline docs/src/rqa/rplots.md:159]: we don't need to save the figures anymore, you can replace all of this with just ending the code block with `fig`

Comment (hkraemer) [reply docs/src/rqa/rplots.md]: 👍

File: docs/src/rqa/rplots.md [hunk 2]
```diff
@@ -116,6 +116,49 @@ which justifies why recurrence plots are so fitting to be used in embedded times
...
+Rg = grayscale(R)
+imshow(Rg, cmap = "binary_r", extent = (1, size(R)[1], 1, size(R)[2]))
+title("RP of monochromatic signal")
+
+subplot(122)
+Rg = grayscale(R_skel)
+imshow(Rg, cmap = "binary_r", extent = (1, size(R)[1], 1, size(R)[2]))
+title("skeletonized RP")
+fig.tight_layout(pad=0.3); fig
+fig # hide
```
Comment (Datseris) [inline docs/src/rqa/rplots.md:157]:
```suggestion
fig
```

**Comment by Datseris** [comment]
@hkraemer I was going to merge that , but I see in the docs 
image

Is `skeletonize` exported in RecurrenceAnalysis.jl? if not, can you please make a PR that exports it and increases patch version in Proejct.toml?

**Comment by hkraemer** [comment]
Hm, it does get exported actually. And there is a docstring.
Bildschirmfoto 2022-04-18 um 11 31 03

**Comment by hkraemer** [comment]
Btw. I will take care of the docs including an appropriate example together with the docs for `TreeEmbedding.jl`, where I am currently trying to fix the tests :)

**Comment by Datseris** [comment]
The documentation has now also switched to using Makie.jl for producing images. I can do this change here if you are not familiar with Makie.jl.

**Comment by hkraemer** [comment]
> The documentation has now also switched to using Makie.jl for producing images. I can do this change here if you are not familiar with Makie.jl.

That would be awesome, thanks.

**Comment by Datseris** [comment]
The docs for whatever reason load version 1.6.3. I'm working on porting to Graphs.jl now.

**Comment by Datseris** [comment]
Looks good :) 

image

**Pull request merged**